### PR TITLE
Alerts Doc: Added section on SES, Formatting #47

### PR DESCRIPTION
**Pull request by paolaperaza**
I added a small section under Sendgrid setup that walks through setting up Airflow email alerts via Amazon's SES.

Otherwise, I:

- Made a few formatting changes
- Adjusted the description (formerly Sendgrid specific)
- Renamed the doc to "Airflow Alerts"

@vparekh94 Since you had previously consolidated this doc from two others that existed prior, can you take a look and just make sure it aligns with what you expected from this doc?

### CODE REVIEW

File: v0.8/setting-up-airflow-emails.md [hunk 1]
```diff
@@ -1,55 +1,117 @@
...
+
+Now, add the following Environment Variables:
+
+```
+AIRFLOW__SMTP__SMTP_HOST=in US-EAST-1, this will be "email-smtp.us-east-1.amazonaws.com
+AIRFLOW__SMTP__SMTP_PORT=587
+AIRFLOW__SMTP__SMTP_STARTTLS=True
+AIRFLOW__SMTP__SMTP_SSL=False
+AIRFLOW__SMTP__SMTP_USER={ENTER_USERNAME_FROM_STEP2A}
+AIRFLOW__S_MTP__SMTP_PASSWORD={ENTER_PASSWORD_FROM_STEP2A}
```
Comment (vparekh94) [inline v0.8/setting-up-airflow-emails.md]: SMTP

File: v0.8/setting-up-airflow-emails.md [hunk 2]
```diff
@@ -1,55 +1,117 @@
...
+AIRFLOW_SMTP__SMTP_PASSWORD={ENTER_SENDGRID_APIKEY_HERE}
+AIRFLOW_SMTP__SMTP_PORT=587
+AIRFLOW_SMTP__SMTP_MAIL_FROM={ENTER_RELEVENT_FROM_EMAIL_HERE}
 ```
 ![Astro Create Envs](https://assets2.astronomer.io/main/docs/emails/astro_create_envs.png)
 
 Click `Update` to save the configuration and redeploy to propagate to your deployment. Your deployment will use that configuration to send emails from then on.
 
-### Triggering Alerts on DAG Run
+## Amazon SES
```
Comment (vparekh94) [inline v0.8/setting-up-airflow-emails.md]: There should be something here that says "If you are using Amazon SES..." - reading it the way it is now almost implies that you need to be using Amazon SES in addition to Sendgrid or something

Comment (paolaperaza) [reply v0.8/setting-up-airflow-emails.md]: Got it, added a lil note

File: v0.8/setting-up-airflow-emails.md (hunk 1 above)
Comment (paolaperaza) [reply v0.8/setting-up-airflow-emails.md]: Oo good catch. Fixed!

**Pull request merged**